src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class RightTriangle extends Figure {
+    public static final int MAXIMUM_SIZE = 50;
+    private int firstLeg;
+    private int secondLeg;
+    private Random random = new Random();
```

Comment:
The Random object should not be a class member of RightTriangle. Random values should be passed to the constructor of the figure instead of generating them inside figure classes. Please remove this and adjust the class to accept leg lengths as parameters to its constructor.